mobile-app/lib/ui/views/news/html_handler/html_handler.dart

```diff
@@ -187,23 +208,59 @@ class HtmlHandler {
               ),
             ),
           );
-        },
-        'blockquote': (code, child) {
+        }),
+        bockQuoteMatcher(): CustomRender.widget(widget: (code, child) {
           return Container(
-            decoration: const BoxDecoration(
-              border: Border(
-                left: BorderSide(
-                    color: Color.fromRGBO(0x99, 0xc9, 0xff, 1), width: 2),
+              decoration: const BoxDecoration(
+                border: Border(
+                  left: BorderSide(
+                      color: Color.fromRGBO(0x99, 0xc9, 0xff, 1), width: 2),
+                ),
               ),
-            ),
-            child: child,
-          );
-        }
+              child: Row(
+                children: [
+                  for (var line in code.tree.element!.text.split('\n'))
+                    Expanded(
+                      child: Padding(
+                        padding: const EdgeInsets.all(8.0),
+                        child: Text(line,
+                            style: TextStyle(
+                                fontSize: code.style.fontSize!.size,
+                                fontFamily: 'lato')),
+                      ),
+                    ),
+                ],
+              ));
+        })
       },
       onLinkTap: (String? url, RenderContext context,
           Map<String, String> attributes, dom.Element? element) {
         launchUrlString(url!);
       },
     );
   }
+
+  static CustomRenderMatcher bockQuoteMatcher() =>
+      (context) => context.tree.element?.localName == 'blockquote';
+
+  static CustomRenderMatcher codeMatcher() =>
+      (context) => context.tree.element?.localName == 'code';
+
+  static CustomRenderMatcher tableMatcher() =>
```

Comment:
Is the custom render for this matcher supposed to be added as I don't find this matcher used in the code above.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Reusable